<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lightweightbaby</class>
 <widget class="QMainWindow" name="lightweightbaby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lightweightbaby</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Добро пожаловать в программу LIGHTWEIGHTBABY!</string>
    </property>
   </widget>
   <widget class="QLabel" name="directoryLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбранная директория: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="finishButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Результаты</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
